### ISSUE TITLE
/home/pi/.gem/gems/telegram-bot-ruby-0.11.0/lib/telegram/bot/client.rb:40: warning: constant ::TimeoutError is deprecated

### ISSUE THREAD

**Issue by phykos** [atipugin/telegram-bot-ruby#210]
I developed a bot for dumping what the bot sees and send it as a JSON file into a chat, and then delete it.
when I runned ruby -C bot.rb it says:
```
Traceback (most recent call last):
/snap/ruby/168/bin/ruby: Can't chdir to bot.rb (fatal)
```

that may be because Ruby 2.7.0 installed from Snap may have a little bit of bugs, but on my phone (Ruby 2.6.5, Termux) it's fine.

that being said, it seems like a Faraday has upgraded its modules and stuff and the library won't actually work

**Comment by ivanovaleksey** [comment]
@phykos what is the version of Faraday in your project?

**Comment by atipugin** [comment]
Message in the title only warns about deprecated const usage. Btw it's been fixed in v0.12.

**Comment by phykos** [comment]
> Message in the title only warns about deprecated const usage. Btw it's been fixed in v0.12.

Thx!
